include/fuse_lowlevel.h

```diff
@@ -1381,6 +1381,19 @@ int fuse_reply_attr(fuse_req_t req, const struct stat *attr,
  */
 int fuse_reply_readlink(fuse_req_t req, const char *link);
 
+/**
+ * Setup passthrough backing file for open reply
+ *
+ * Possible requests:
+ *   open, opendir, create
+ *
+ * @param req request handle
+ * @param fd backing file descrptor
```

Comment:
Probably want to fix this typo resulting in static analysis failure while waiting for actual maintainers to review.

Reply:
done